src/ft2font_wrapper.cpp

```diff
@@ -254,7 +255,7 @@ static PyTypeObject *PyGlyph_init_type(PyObject *m, PyTypeObject *type)
  * FT2Font
  * */
 
-typedef struct
+struct PyFT2Font
```

Comment:
there seemed to be a cross dependency.. see `forward.hpp`

Reply:
I was getting: 
```console
warning: ‘typedef’ was ignored in this declaration
 typedef struct PyFT2Font
```

Reply:
If you're asking why did I specifying PyFT2Font here.. it's because it's needed in FT2Font class.. there was cross dependency (also see `forward.hpp`)